nautobot/core/app/__init__.py

```diff
@@ -0,0 +1,244 @@
+from django.apps import AppConfig
```

Comment:
Per Jathan's comment, this directory should be `nautobot/core/apps/` not `nautobot/core/app`.